src/promptflow-core/promptflow/contracts/flow.py

```diff
@@ -401,6 +401,68 @@ def deserialize(data: dict) -> "FlowInputDefinition":
         )
 
 
+class InitParamType(str, Enum):
+    """Value types."""
+
+    AZURE_OPEN_API_MODEL_CONFIGURATION = "AzureOpenAIModelConfiguration"
+    OPEN_AI_MODEL_CONFIGURATION = "OpenAIModelConfiguration"
+
+    AZURE_OPEN_AI_CONNECTION = "AzureOpenAIConnection"
```

Comment:
It looks like we need to add new line here when introduce new connection type. 
Can we avoid this?